UI/src/view/navigationpanel/LinearAssetBox.js

```diff
@@ -31,7 +31,11 @@
     var bindExternalEventHandlers = function() {
       eventbus.on('roles:fetched', function(roles) {
         userRoles = roles;
-        if (_.contains(roles, 'operator') || _.contains(roles, 'premium')) {
+        if (_.contains(roles, 'operator') || (_.contains(roles, 'premium') && !elements.expanded.hasClass('panel maintenanceRoad'))) {
```

Comment:
we Can join this IF clause with ELSE IF clause below?
I say that, because the IF clause and the ELSE IF clause do the same code.